tests/layer_tests/ovc_python_api_tests/test_pytorch.py

```diff
@@ -1093,6 +1093,21 @@ def test_example_inputs(self):
         with self.assertRaisesRegex(TypeError, ".*got an unexpected keyword argument 'example_inputs'.*"):
             convert_model(pytorch_model, example_inputs=(torch.tensor(1),))
 
+    def test_incorrect_inputs_1(self):
+        from openvino.tools.ovc import convert_model
+        pytorch_model, _, _ = create_pytorch_nn_module_case1('')
+
+        with self.assertRaisesRegex(Exception, ".*No node with name.*"):
+            convert_model(pytorch_model, input='input1[1, 10]')
+
+    def test_incorrect_inputs_2(self):
+        from openvino.tools.ovc import convert_model
+        pytorch_model, _, _ = create_pytorch_nn_module_case1('')
+
+        # check that it accepts specified names as is without parsing into 2 different inputs
+        with self.assertRaisesRegex(Exception, 'No node with name input1\[1, 10\],input2\[2, 100\]'):
+            convert_model(pytorch_model, input='input1[1, 10],input2[2, 100]')
```

Comment:
Please add negative test for
convert_model(pytorch_model, input='input1,input2')

Reply:
And this negative test:
convert_model(pytorch_model, input='input1[1,10]','input2[2,100]')

Reply:
done

Reply:
added test for `convert_model(pytorch_model, input=['input1[1,10]','input2[2,100]'])` as well.